### ISSUE TITLE
[Bug]when consumer has early than provider started, consumer will throw NPE..

### ISSUE THREAD

**Issue by bearDream** [apache/incubator-dubbo-spring-boot-project#126]
If I use this project in product enviroment, this bug can bring some troubles. I think if consumer can't find provider, only log it, don't neet throw NPE.

**Comment by sarins** [comment]
I have got same error, how to ?

**Comment by sarins** [comment]
@mercyblitz Which version we will plan to fix this bug ? and is this issue same with #187?

**Comment by mercyblitz** [comment]
Please add dependencies in your Maven `pom.xml`, like this:

```xml
<!-- Dubbo -->
<dependency>
    <groupId>com.alibaba</groupId>
    <artifactId>dubbo</artifactId>
    <version>2.6.5</version>
</dependency>
<!-- Spring Context Extras -->
<dependency>
    <groupId>com.alibaba.spring</groupId>
    <artifactId>spring-context-support</artifactId>
    <version>1.0.2</version>
</dependency>
```
